Last-Update: 2011-08-06
Description: New path for referenced files
  Debian policy requires documentation and configuration files to lie
  in a place different than used by upstream.

diff --git a/html2ps b/html2ps
index 1404164..45351d2 100755
--- a/html2ps
+++ b/html2ps
@@ -24,8 +24,8 @@
 # Set the name of the global configuration file. See the installation notes
 # and manual page for more details on configuration files.
 
-$globrc='/it/sw/share/www/lib/html2ps/html2psrc';
-$ug='/it/sw/share/www/lib/html2ps/html2ps.html';
+$globrc='/etc/html2psrc';
+$ug='/usr/share/doc/html2ps/html2ps.html';
 $default_paper=`paperconf` || 'a4';
 
 $conf=<<"EOR";
